packages/devtools_app/lib/src/debugger/flutter/debugger_screen.dart

```diff
@@ -98,11 +99,22 @@ class DebuggerScreenBodyState extends State<DebuggerScreenBody>
     await controller.selectScript(ref);
   }
 
+  Future<void> _onBreakpointSelected(BreakpointAndSourcePosition bp) async {
```

Comment:
can the controller maintain the selected breakpoint using the value notifier pattern instead of the view?

Reply:
the selected breakpoint probably doesn't need to be on the controller as that seems to be purely view state.

Reply:
ok, gotcha